dll/win32/shell32/dialogs/drive.cpp

```diff
@@ -411,8 +411,6 @@ InitializeFormatDriveDlg(HWND hwndDlg, PFORMAT_DRIVE_CONTEXT pContext)
     SendMessageW(hwndFileSystems, CB_SETCURSEL, dwDefault, 0);
     /* setup cluster combo */
     InsertDefaultClusterSizeForFs(hwndDlg, pContext);
-    /* hide progress control */
-    ShowWindow(GetDlgItem(hwndDlg, 28678), SW_HIDE);
```

Comment:
Maybe in a separate commit (before this one) you may try to un-hardcode this resource value :)

Reply:
Ok just forget about the "before this one" part.

Reply:
I'd be more than happy to open a new PR that un-hardcodes the format dialog. :) Maybe after this is committed?